src/_experimental_/GraphQL/QlHooks.re

```diff
@@ -513,4 +583,4 @@ let useIsForeclosed: Web3.ethAddress => bool =
     optAvailableDeposit->Option.mapWithDefault(true, availableDeposit =>
       availableDeposit->BN.ltGet(. BN.new_("0"))
     );
-  };
+  };
```

Comment:
I'll add a `.vscode` config to this repo so that we all have the same config for tailing newlines.